.github/profile/README.md

```diff
@@ -0,0 +1,107 @@
+ <img src="https://github.com/user-attachments/assets/e3e706de-2225-4242-84cd-a9dbe517c447" />
+
+
+**Welcome to the [ORC](https://orcwg.org/) GitHub space** – an [Eclipse Foundation Working Group](https://eclipse.org/) working on supporting Open Source Community to be compliant with the demands of the *[Cyber Resilient Act](https://digital-strategy.ec.europa.eu/en/policies/cyber-resilience-act).* 
```

Comment:
A few comments:

- I think Github calls this an _organization_ and that _(code)spaces_ are something else.
- wouldn't add links to external resources so high up in the page, as you risk having people navigate away instead of following the links on how to contribute.

```suggestion
**Welcome to the ORC GitHub organization** – ORC is an Eclipse Foundation Working Group whose immediate focus is to help open source maintainers and users meet the compliance requirements of the **European Cyber Resilience Act (CRA).**
```

Maybe we can add the links back down somewhere?